addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3433,8 +3433,9 @@ registry.ImageOptimize = ImageHandlerOption.extend({
      * @override
      */
     _computeMaxDisplayWidth() {
+        // 128 is the minimum width that we can suggest to avoid bad rendering
         // TODO: read widths from computed style in case container widths are not default
-        const displayWidth = this._getImg().clientWidth;
+        const displayWidth = Math.max(this._getImg().clientWidth, 128);
```

Comment:
If we have to set a higher limit, it should be way higher than this

See 
![image](https://user-images.githubusercontent.com/30048408/124140355-b0b52100-da88-11eb-891b-5130f72fa915.png)

Still blurry
Better with higher number
![image](https://user-images.githubusercontent.com/30048408/124140457-c9bdd200-da88-11eb-9114-3375ed73da3e.png)


Also, I'd expect the logo to fit the full width of the menu

Reply:
After thinking about this, I think that we should indeed not hardcode a fixed limit, as the problem is visible as soon as there is too big a difference between the original logo width and the suggested resize width. Therefore as a temporary fix maybe we could set the limit of the suggested width as the `originalWidth / 2`, but the real fix would be to change the resize method, which would take a bit more time. I am trying to render with webgl to see if it makes a difference

Reply:
To keep you posted I tried with Hermite resize, the results seem to be slightly better but not good enough

Hermite resize :
![image](https://user-images.githubusercontent.com/58561718/124788732-ee62ef80-df49-11eb-8984-624cface4daf.png)
Legacy Resize :
![image](https://user-images.githubusercontent.com/58561718/124788942-20745180-df4a-11eb-8ac5-14b695d6c97a.png)

I will continue to investigate ways to resize smoothly from the browser, otherwise we can fallback to limit the resize to a 1:2 ratio :
![image](https://user-images.githubusercontent.com/58561718/124789528-a98b8880-df4a-11eb-81bc-de5da6f28279.png)
Or even to 1:3 :
![image](https://user-images.githubusercontent.com/58561718/124789680-cd4ece80-df4a-11eb-984c-dc360f20c9ad.png)
But 1:4 seems to be too degrading already :
![image](https://user-images.githubusercontent.com/58561718/124789884-f4a59b80-df4a-11eb-90d8-44ad78f0311a.png)

Reply:
Lastly, I tried the pica js library https://github.com/nodeca/pica which gives pretty good results for client side resizing :
![image](https://user-images.githubusercontent.com/58561718/124916279-c7acc380-dff2-11eb-9990-d38040e64a34.png)

The cons are that it is less straightforward to integrate/maintain and the issue with the logos will still be there if the user zooms a bit in the page, or has a browser at 125% for better reading for example.

So I think that we will need to limit the suggested width for optimization in any case.
The question would be : do we need to integrate pica to odoo or, if we solve that logo issue with a limit on the suggested width, there is no such need ?